app/models/storage_location.rb

```diff
@@ -83,7 +83,7 @@ def self.import_csv(csv, organization)
   # NOTE: We should generalize this elsewhere -- Importable concern?
   def self.import_inventory(filename, org, loc)
     current_org = Organization.find(org)
-    adjustment = current_org.adjustments.create(storage_location_id: loc.to_i, comment: "Starting Inventory")
+    adjustment = current_org.adjustments.create(storage_location_id: loc.to_i, user_id: current_org.users.find_by(organization_admin: true)&.id, comment: "Starting Inventory")
```

Comment:
I actually hadn't considered this when I wrote the ticket initially, so good find!